pyproject.toml

```diff
@@ -1,10 +1,11 @@
 [project]
-name = "google-tasks-api"
+name = "tasks-board"
 version = "0.1.0"
 description = "Add your description here"
 readme = "README.md"
 requires-python = ">=3.10"
 dependencies = [
     "google-api-python-client>=2.166.0",
     "google-auth-oauthlib>=1.2.1",
```

Comment:
why are these needed for the interface?

Reply:
Removed. I was trying an implementation on the side, this made it through from those changes.